inc/stats/namespace.php

```diff
@@ -492,7 +492,7 @@ function display_network_storage() {
 }
 
 function calculate_network_storage() {
-	$path = wp_upload_dir()['basedir'];
+	$path = realpath( wp_upload_dir()['basedir'] );
 	$storage = format_bytes( rtrim( str_replace( $path, '', `du -b -s $path` ) ) );
```

Comment:
$path can theoretically be false.

$path can theoretically be exploited (the command line version of https://xkcd.com/327/) it should be wrapped in escapeshellarg()